packages/upload-client/src/sharding.js

```diff
@@ -0,0 +1,112 @@
+import Queue from 'p-queue'
+import { encode } from './car.js'
+import { add } from './store.js'
+
+// most thing are < 30MB
```

Comment:
sure

Reply:
> in general, when your object size reaches 100 MB, you should consider using multipart uploads instead of uploading the object in a single operation.
– https://docs.aws.amazon.com/AmazonS3/latest/userguide/mpuoverview.html

Reply:
Ok I update to 100